pages/documentation.html

```diff
@@ -30,50 +37,150 @@
 
 </div>
 
+<div class="framework">
+<div class="frameworklabel">Books and long tutorials</div>
+<div class="frameworkcontent">
 
+<p>The following books and long tutorials were written by experimented
+Coq users and teachers:</p>
 
-<div class="framework">
-<div class="frameworklabel">Other documentation</div>
+<ul>
 
-<div class="frameworkcontent">
-<p>Moreover, some users of Coq have published various documents that may be useful both to beginners and advanced users of Coq:</p> 
+<li>the <a href="http://www.labri.fr/perso/casteran/CoqArt/">Coq'Art</a>,
+the first book dedicated to the Coq proof assistant (Yves Bertot,
+Pierre Castéran, 2004, Chinese version in 2009), only the French
+version and the exercises can be downloaded for free;</li>
+
+<li><a href="http://www.cis.upenn.edu/~bcpierce/sf/">Software
+Foundations</a>, a series of Coq-based textbooks on logic, functional
+programming and foundations of programming languages (Benjamin
+Pierce <em>et al</em>, 2007, with regular updates), much acclaimed for
+being accessible to beginners, but rather oriented to computer
+scientists;</li>
+
+<li><a href="http://adam.chlipala.net/cpdt/">Certified Programming
+with Dependent Types</a>, a textbook about practical engineering with
+Coq (Adam Chlipala, 2008), teaches advanced practical tricks and a
+very specific style of proof;</li>
+
+<li>the <a href="https://math-comp.github.io/mcb/">Mathematical
+Components book</a> (A. Mahboubi and E. Tassi, 2018), a book that is
+more oriented to mathematical inclined users, to dive into Coq with
+the SSReflect proof language, and the Mathematical Components
+library.</li>
+
+<!-- TODO Add FRAP (Chlipala) and PnP (Sergey) -->
+
```

Comment:
It may be worth mentioning Boldo & Melquiond's [Computer Arithmetic and Formal Proofs](http://iste.co.uk/book.php?id=1238), which is based on Coq but domain specific. Some academic can download the PDF without cost via [Science Direct](https://www.sciencedirect.com/book/9781785481123/computer-arithmetic-and-formal-proofs).

Reply:
I'll just go ahead and do a suggestion:
```suggestion
<li><a href="http://iste.co.uk/book.php?id=1238">Computer Arithmetic and Formal Proofs</a>, (S. Boldo and G. Melquiond, 2017), a book that gives
a comprehensive view of how to formally specify and verify tricky floating-point algorithms with Coq using the
<a href="https://gitlab.inria.fr/flocq/flocq">Flocq library</a>.</li>
```